src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java

```diff
@@ -30,6 +30,7 @@
 import javax.annotation.Nullable;
 
 import org.joda.time.Period;
+import org.jooq.tools.StringUtils;
```

Comment:
Apache Commons StringUtils not available ?

Reply:
Updated.